modules/npcs/src/main/java/eu/cloudnetservice/modules/npc/platform/bukkit/command/NPCCommand.java

```diff
@@ -614,6 +666,7 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
       return Arrays.asList(
         "create",
         "edit",
+        "info",
```

Comment:
Info is now tab-completable but it's not listed in the help output. Is this command more of a debug thing that shouldn't be listed? In that case I would remove it from tab completion as well.